crs_application/config/localization.yaml

```diff
@@ -23,14 +23,14 @@ sac:
   inlier_fraction: 0.25
 crop_boxes:
   box1:
-    xyz: [-0.4, 0.0, 1.15]
-    size: [2.0, 2.0, 0.8]
+    xyz: [0.0, -0.875, -0.25]
+    size: [2.0, 0.4, 0.2]
```

Comment:
Is this specific to the test workcell?

Reply:
Yes, if there another way I can change these parameters without modifying this file?

Reply:
What I was thinking to do for handling  multiple workcell with their own separate set of configuration files is to create separate directories where each one hosts the config files for a specifc workcell, so we for instance in **crs_application/config** we could have a directory structure like this
crs_application/config/main
- crs.yaml
- part_localization.yaml
  .
  .
crs_application/config/surrogate_1
- crs.yaml
- part_localization.yaml
  .
  .

where main has the files for the original workcel and surrogate_1 has the files for the most recent one, We'd have to parameterize the directory locations in the yaml files but that shouldn't be too hard

Reply:
For now I'm ok if we at least leave them commented out since we'll have to restore them for the wsu setup

Reply:
Okay, they will likely also change depending on the new part height at WSU